hack/test-cmd.sh

```diff
@@ -257,6 +251,9 @@ echo "config files: ok"
 export KUBECONFIG="${HOME}/.kube/non-default-config"
 export CLUSTER_ADMIN_CONTEXT=$(oc config view --flatten -o template -t '{{index . "current-context"}}')
 
+pushd "$OS_ROOT" > /dev/null
+tests=( "${1:-test/cmd}"/*.sh )
+popd > /dev/null
```

Comment:
I would simply use `tests=( "${OS_ROOT}/${1:-test/cmd}"/*.sh )`, but I am assuming that we want to allow `$1` to be an absolute path.